src/pages/home/report/ReportActionItem.js

```diff
@@ -181,33 +265,25 @@ class ReportActionItem extends Component {
                                     isMini
                                 />
                             </View>
-                            <PopoverWithMeasuredContent
-                                isVisible={this.state.isPopoverVisible}
-                                onClose={this.hidePopover}
-                                anchorPosition={this.popoverAnchorPosition}
-                                animationIn="fadeIn"
-                                animationOutTiming={1}
-                                measureContent={() => (
-                                    <ReportActionContextMenu
-                                        isVisible
-                                        selection={this.selection}
-                                        reportID={this.props.reportID}
-                                        reportAction={this.props.action}
-                                        hidePopover={this.hidePopover}
-                                    />
-                                )}
-                            >
-                                <ReportActionContextMenu
-                                    isVisible
-                                    reportID={this.props.reportID}
-                                    reportAction={this.props.action}
-                                    draftMessage={this.props.draftMessage}
-                                    hidePopover={this.hidePopover}
-                                />
-                            </PopoverWithMeasuredContent>
                         </View>
                     )}
                 </Hoverable>
+                <PopoverWithMeasuredContent
+                    isVisible={this.state.isPopoverVisible}
+                    onClose={this.hidePopover}
+                    anchorPosition={this.state.popoverAnchorPosition}
+                    animationIn="fadeIn"
+                    animationOutTiming={1}
+                    measureContent={this.measureContent}
+                >
+                    <ReportActionContextMenu
+                        isVisible
+                        reportID={this.props.reportID}
+                        reportAction={this.props.action}
+                        draftMessage={this.props.draftMessage}
+                        hidePopover={this.hidePopover}
+                    />
+                </PopoverWithMeasuredContent>
```

Comment:
This context menu is used for each message on the chat. But when we hover over a message, it will never change its position then re-rendering it is completely unnecessary.